frontend/packages/knative-plugin/integration-tests/features/admin-perspective/eventing-page-admin.feature

```diff
@@ -5,6 +5,8 @@ Feature: Eventing page at Administrator perspective
 
         Background:
             Given user is at administrator perspective
+              And user has installed OpenShift Serverless Operator
+              And user has deployed Knative Eventing component
```

Comment:
Actually we have included the [code](https://github.com/openshift/console/blob/master/frontend/packages/knative-plugin/integration-tests/support/commands/hooks.ts#L8) for knative installation in before hook, because all the scenarios present in Knative plugin this installation is mandatory

For End user understanding purpose, If you are adding these, Lets keep it, but we can comment it out